internal/middlewares/access_managment.go

```diff
@@ -0,0 +1,180 @@
+package middlewares
```

Comment:
please fix typo in file name.

Reply:
Done

Reply:
Just to clarify, I was pointing out to missing `E`: `access_managEment`. Manager is fine also.